.github/workflows/test-with-pypi.yml

```
@@ -0,0 +1,40 @@
+name: Test with PyPI
+
+on: [pull_request, workflow_dispatch]
+
+env:
+  PYTHONUNBUFFERED: 1
+
+jobs:
+
+  # Test against PyPI packages
+  test-with-pypi:
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        python-version: ['3.11']
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Clone the Scimath source
```

Nitpick: the name is `SciMath`
```suggestion
      - name: Clone the SciMath source
```